client/src/common/components/DescriptionTextarea.jsx

```diff
@@ -52,6 +53,7 @@ class DescriptionTextarea extends PureComponent {
 
 DescriptionTextarea.propTypes = {
   description: PropTypes.string.isRequired,
+  disabled: PropTypes.bool.isRequired,
```

Comment:
shouldn't be required - by default it is false. Otherwise we will have to always set prop `disabled={false}`